src/Requests/Traits/PaymentRequestTrait.php

```diff
@@ -0,0 +1,271 @@
+<?php
+
```

Comment:
`declare(strict_types = 1);` in all places (includes tests)

Reply:
Fixed